### DISCUSSION TITLE
Native "Optimized Battery Charging"

### DISCUSSION THREAD

**Discussion by dedalnok** [rurza/BatFi#64] [General]
Should I disable the native "optimized battery charging" from Apple in the battery health menu?
![Screenshot 2024-07-07 at 14 25 08](https://github.com/rurza/BatFi/assets/174910671/e3f44919-c9ec-4c72-8f27-5b73bfb0265a)

**Comment by holocronology** [comment]
I was just wondering the same thing. No answer to this as yet, eh?
